litmus-portal/frontend/src/components/DateTime/CustomTime/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { MuiPickersUtilsProvider, TimePicker } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './styles';
+
+interface CustomTimeProps {
+  ampm: boolean;
+  disabled: boolean;
+}
+
+// Used to set and display time in hours and minutes
+const CustomTime: React.FC<CustomTimeProps> = ({ ampm, disabled }) => {
+  const classes = useStyles();
+  const [selectedDate, setSelectedDate] = React.useState<Date | null>(
+    new Date(Date.now())
+  );
+
+  const handleDateChange = (date: Date | null) => {
+    setSelectedDate(date);
+  };
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <TimePicker
+        className={classes.timePicker}
+        ampm={ampm}
+        disabled={disabled}
+        inputVariant="outlined"
+        inputProps={{
+          style: {
+            fontSize: '0.75rem',
+            color: '#000000',
+            paddingLeft: 29,
```

Comment:
For the padding 'rem' can be used if not spacing from theme. (Avoid using absolute units). @Saranya-jena